layouts/shortcodes/min-credits-report.md

```diff
@@ -1,2 +1,2 @@
-We'll write a report for pentests with at least eight (8) credits
+We'll write a report for pentests with at least five (5) credits
```

Comment:
Full report is available with compliance tests only.
Note to DOCS Team: Do you want to add about Agile PT report as well?

Reply:
Changed to:

We'll write a Full Report for Compliance Pentests<!--. An Automated Report is available for Agile Pentests-->

Info about Agile Pentests is commented out before Oct 3 release.